.github/workflows/build-pr.yml

```diff
@@ -25,6 +25,8 @@ on:
     branches:
       - main
   workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
```

Comment:
what is the purpose for that ?

Reply:
Run CI with schedule  for discover flack tests early, does make sence?

Reply:
this workflow only runs on PRs so I don't think it makes much sense

Reply:
Moved schedule to main workflow,and testing for run snapshot exclude schedule.